src/components/Pages/MainPage/story.jsx

```diff
@@ -10,8 +12,17 @@ export default {
     },
   },
 };
-
 const Template = (args) => <Component {...args} />;
 
 export const Default = Template.bind({});
 Default.args = {};
+Default.decorators = [
+  (Story) => (
+    <div>
+      <AlertHandlerProvider>
+        <AlertManager />
+        <Story />
+      </AlertHandlerProvider>
+    </div>
```

Comment:
```suggestion
    <>
      <AlertHandlerProvider>
        <AlertManager />
        <Story />
      </AlertHandlerProvider>
    </>
```
reactのFragmentという機能を使うと、このように書くことができます。
こうすると、余計なDOM要素を加えなくて済むのでおすすめです。